src/lib/generator.js

```
@@ -307,12 +317,7 @@ class Generator {
   }
 
   isMatchingPrimaryImg(containersLen, resourceName, isPrimary) {
-    if (containersLen <= 1) {
-      if (containersLen <= 0) {
```

Don't you still want to throw an error in the case there are no containers AND no `isPrimary` set?

---

0 containers is impossible(the func is inside the `for range containers.lengh`) and if we have more than one without primary should  throw an err